tests/integration-tests/configs/develop.yaml

```diff
@@ -124,21 +125,21 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
     test_cluster_custom_resource.py::test_cluster_update_tag_propagation:
       dimensions:
-        - oss: [ "alinux2" ]
+        - oss: [ "ubuntu2004" ]
           regions: ["us-east-2"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
     test_cluster_custom_resource.py::test_cluster_delete_out_of_band:
       dimensions:
-        - oss: ["alinux2"]
+        - oss: ["ubuntu2204"]
           regions: ["us-east-2"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
     test_cluster_custom_resource.py::test_cluster_delete_retain:
       dimensions:
-        - oss: ["alinux2"]
+        - oss: ["rocky8"]
           regions: ["us-east-2"]
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
     test_cluster_custom_resource.py::test_cluster_create_with_custom_policies:
       dimensions:
-        - oss: ["alinux2"]
+        - oss: ["rhel8"]
```

Comment:
did you mean rocky8 here?

Reply:
This was to address the change Enrico Suggested. https://github.com/aws/aws-parallelcluster/pull/5728#discussion_r1345866850